src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("d.MM.yyyy");
+    private static final int DATE_IND = 0;
+    private static final int NAME_IND = 1;
+    private static final int HOUR_QUANTITY_IND = 2;
+    private static final int SALARY_PER_HOUR_IND = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder res = new StringBuilder();
```

Comment:
[Checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#remember-about-informative-names-of-the-variables)